linux_os/guide/system/accounts/accounts-restrictions/root_logins/ensure_pam_wheel_group_empty/bash/shared.sh

```diff
@@ -7,4 +7,4 @@ if ! grep -q "^${var_pam_wheel_group_for_su}:[^:]*:[^:]*:[^:]*" /etc/group; then
 fi
 
 # group must be empty
-groupmems -g ${var_pam_wheel_group_for_su} -p
+sed -i -E "s/^(${var_pam_wheel_group_for_su}:[^:]*:[^:]*:)[^:]*/\1/g" /etc/group
```

Comment:
What is your opinion on using gpasswd instead of sed directly to edit the group file, say:
```
gpasswd -M '' ${var_pam_wheel_group_for_su}
```

Reply:
It should work fine for rhel. It is fine for Ubuntu too @dodys ? If so I would prefer to use the command suggested by @teacup-on-rockingchair instead of directly editing the file.

Reply:
Hi @teacup-on-rockingchair @marcusburghardt 
I checked and it works fine on Ubuntu 20.04 and 22.04.

Reply:
Great. I will update it now.

Reply:
Done. Thanks @teacup-on-rockingchair and @mpurg